tests/utils/log/test_es_task_handler.py

```diff
@@ -262,21 +277,55 @@ def test_set_context_w_json_format_and_write_stdout(self):
         self.es_task_handler.json_format = True
         self.es_task_handler.set_context(self.ti)
 
-    def test_close(self):
-        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
```

Comment:
i think we should not remove this test as it is still a valid test case with the formatter set

Reply:
and your test only focus on `json format`

Reply:
as I mentioned non-json format does not really work in the context of elastic search. I suspect no one can deploy it correctly.

Reply:
sorry, i did not get it, `non-json format does not really work in the context of elastic search`. We don't use json format and it works fine in our production. 

could you setup be related to this? https://github.com/apache/airflow/pull/7199

Reply:
This comment seems to still be an issue @larryzhu2018 - pingzh is saying that they use ElasticSearch logs without the JSON format enabled.

Reply:
@pingzh you do not use log_id at all in your deployment. you need that in order to make the airflow webui to work as the web ui is searching for the log_id, without that it will keep spinning. Can you please review the comments I added here and check? 

please come back to me here, and tell us if you can make this work with the non-json log format. I do not think you can construct the log_id that is needed for the web-ui using the non-json log format.
Let me recap: IIUC the only reason that you think your deployment worked was because you missed out the log_id logic completely and I have explained how it should work, where it is used, and also add added a unit test case to show you how this can work. Please reviews my explanations here and see if it is clear to you now. Thanks.